### ISSUE TITLE
Typo in 

### ISSUE THREAD

**Issue by codeeesm** [aymericdamien/TensorFlow-Examples#133]
https://github.com/aymericdamien/TensorFlow-Examples/blob/master/examples/3_NeuralNetworks/convolutional_network.py#L70

Should line 70 code changed from 
"fc1 = tf.nn.dropout(fc1, dropout)" to "fc1 = tf.nn.dropout(fc1, keep_prob)"

Otherwise during evaluation, dropout is still in action.

Just my 2 cents as a new comer in ML.
